qa/tasks/scripts.py

```diff
@@ -0,0 +1,394 @@
+from util import (
+    remote_run_script_as_root,
+    )
+
+proposals_dir = "/srv/pillar/ceph/proposals"
+
+
+class Scripts:
+
+    script_dict = {
```

Comment:
I suggest to get rid of this dictionary, in store scripts close to the scripts.py file, for example, directory structure can look like:
```
qa/tasks/scripts.py
qa/tasks/scripts/ceph_cluster_status.sh
a/tasks/scripts/ceph_version_sanity.sh.sh 
qa/tasks/scripts/create_all_pools_at_once.sh.sh 
qa/tasks/scripts/custom_storage_profile.sh.sh 
qa/tasks/scripts/mgr_dashboard_module_smoke.sh.sh 
qa/tasks/scripts/mgr_plugin_influx.sh.sh 
qa/tasks/scripts/proposals_remove_storage_only_node.sh.sh 
qa/tasks/scripts/rados_write_test.sh.sh 
qa/tasks/scripts/remove_storage_only_node.sh.sh 
qa/tasks/scripts/rgw_init.sh.sh 
qa/tasks/scripts/rgw_init_ssl.ssh.sh 
qa/tasks/scripts/salt_api_test.sh.sh
```
Correspondingly the function *remote_run_script_as_root* should be part of Script class and look like:
```
def remote_run_script_as_root(remote, name):
     local_path = os.path.dirname(os.path.realpath(__file__)) + '/scripts/' + name
     .
     .
     .
``` 
The benefits are:
1) Better coding style.
2) Easier to test run the scripts separately from the python code.

Reply:
Good idea. Created https://trello.com/c/sTIp68ze/275-ses6-optimize-scriptspy to track it.